tests/unittests/general/prefix_ip_collector_test.py

```diff
@@ -14,5 +14,5 @@ def test_find_range(self):
         self.assertEqual(find_range('2001:700:0:251e::/64'), 0)
 
     def test_find_timestamp(self):
-        ts = datetime(2012, 10, 4, 14, 30)
-        self.assertEqual(get_timestamp(ts), 1349353800)
+        ts = datetime(2012, 10, 4, 14, 30, tzinfo=timezone.utc)
+        self.assertEqual(get_timestamp(ts), 1349361000)
```

Comment:
You should also have a test that checks what get_timestamp() does if given a naive datetime. (And probably ensure that that specific test runs under its own, non-utc timezone. It might be relevant to use `freezegun` or similar.)

Reply:
In `django.utils.timezone` there is a context-manager "override" that should be perfect for the job!

Reply:
I could agree, but I'm not sure how relevant it is for this particular PR, which is about fixing a broken test (the slight `get_timestamp` *refactor* was just a bonus), not adding more ones.

Naive `datetime` objects will return a unix timestamp as if they were defined as UTC already, and I'd just be testing Python's datetime behavior here, not `get_timestamp`'s.

That being said, `get_timestamp()` will really only ever be fed naive timestamps - they all come from PostgreSQL queries that produce tz-naive timestamps (unfortunately).

Reply:
This is testing as documentation though. Handing date and time is tricky no matter the language, so having proof of what is supposed to happen is always good.